apps/ml/docker-compose.yml

```diff
@@ -0,0 +1,9 @@
+version: '3'
+services:
+  core_api:
+    build: .
+    container_name: "core-api-container"
+    ports:
+      - "8000:15400"
+    volumes:
+      - ./app/:/app
```

Comment:
Lets also modify this a little bit, so add the following:

```
command: uvicorn app.main:app --host 0.0.0.0 --port 15400 --reload
restart: unless-stopped
```
so we have hot reload for dev and auto container restart